src/sentry/static/sentry/app/components/eventsTable/eventsTableRow.jsx

```diff
@@ -76,7 +58,7 @@ class EventsTableRow extends React.Component {
             <GlobalSelectionLink to={link}>
               <DateTime date={event.dateCreated} />
             </GlobalSelectionLink>
-            <small>{(this.getEventTitle(event) || '').substr(0, 100)}</small>
+            <small>{event.title.substr(0, 100)}</small>
```

Comment:
This matches how `app/components/eventOrGroupTitle` is implemented now too.

Reply:
Good catch, will tackle that one too.

Reply:
Actually that one might have to stay for now, we need a subtitle as well there.